### ISSUE TITLE
[A11yInspect] Fail: 4.1.2 A in /home/runner/work/test-app/test-app/src/App.tsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#21]
## ❌ Accessibility Fail

**Rule:** 4.1.2 A
**File:** `/home/runner/work/test-app/test-app/src/App.tsx`
**Line:** 38, **Column:** 1

### Description
Missing accessible name for button (Critical)

### Context
```html
<button></button>
```

### Selector
`html > body > div > audio > button:nth-of-type(1)`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
